package/zshelf/package

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(zshelf)
+pkgdesc="Z-Library browser and downloader"
+url=https://github.com/khanhas/zshelf
+pkgver=0.1.1
```

Comment:
```suggestion
pkgver=0.1.1-1
```

This adds the package version. It is separate from your software version in the github repo and indicates how often we packaged that particular software version. When for example some repo changes affect your package or the packaging changes (e.g. a hook/script change or what files get included) this last part gets increased (either by the toltec maintainers (!) your yourself) to indicate an update in packaging but not the actual software itself.

An example of us increasing this number due to changes of many packages metadata can be seen [here](https://github.com/toltec-dev/toltec/pull/226/files).